README.md

```diff
@@ -8,21 +8,21 @@
 
 ## Why is WordPress Playground useful?
 
-WordPress Playground exists to make WordPress instantly accessible for users, learners, extenders, and contributors by building foundational software tools developers can use to create interactive, zero-setup, JavaScript applications with WordPress.
+WordPress Playground aims to make WordPress instantly accessible for users, learners, extenders, and contributors. It does this by providing foundational software tools that enable developers to create interactive, zero-setup JavaScript applications with WordPress.
```

Comment:
> @brandonpayton was it an unsubmitted review by any chance? I don't see the nit

@adamziel, that's strange. I even have a vague memory of choosing to click the "Start a Review" button for that comment.

@fellyph the nit was something like:

I feel like the word "exists" is clearer than "aims" at communicating the purpose of the project. I like that "exists" suggests that the project wouldn't even exist without the goal. It seems a little deeper than "aims" which says to me that "a preexisting thing now has a goal".

☺️ Honestly, either should be fine, but I didn't think there was a problem with the previous word and liked it for the reason above.